dandischema/consts.py

```diff
@@ -11,6 +11,7 @@
     "0.6.5",
```

Comment:
I think `DANDI_SCHEMA_VERSION = "0.6.8"` in line 1 should change to `DANDI_SCHEMA_VERSION = "0.6.9" as well since we are publishing a new version of the schema.